pkgs/by-name/vk/vkdevicechooser/package.nix

```
@@ -0,0 +1,58 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  writeText,
+  meson,
+  vulkan-headers,
+  vulkan-validation-layers,
+  ninja,
+  jq,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "vkdevicechooser";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "aejsmith";
+    repo = "vkdevicechooser";
+    rev = version;
+    hash = "sha256-low5vkuZT4gWta4PeoCaP/jIO1uszACq+LTZopo8BAI=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    jq
+  ];
+
+  buildInputs = [
+    vulkan-headers
+    vulkan-validation-layers
+    ninja
+  ];
+
+  # Help vulkan-loader find the validation layers
+  setupHook = writeText "setup-hook" ''
+    addToSearchPath XDG_DATA_DIRS @out@/share
+  '';
+
+  mesonFlags = [ "-Dc_args=-I${vulkan-validation-layers}/include" ];
```

```suggestion
  mesonFlags = [ 
    (lib.mesonOption "c_args" "-I${lib.getInclude vulkan-validation-layers}/include")
  ];
```

P.S.: do not forget to run formatter.